linkerd/meshtls/boring/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "linkerd-meshtls-boring"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+license = "Apache-2.0"
+edition = "2018"
+publish = false
+
+[dependencies]
+boring = { version = "1" }
+boring-sys = { version = "1"}
```

Comment:
do we actually need to depend on the `sys` crate here? i assumed that would be a transitive dependency of `boring`, and i don't see anywhere where we actually use anything from the `sys` crate?